Bruno Collection - CDS Videos Publish Video.json

```diff
@@ -4,22 +4,22 @@
   "items": [
     {
       "type": "http",
-      "name": "Optional Change access of the video",
-      "seq": 8,
+      "name": "Step 6: Get Project to Check the Flow Status",
+      "seq": 7,
```

Comment:
shouldn't be this 6 or should you update the description of the step?

Reply:
I just changed the order of requests. `Optional: Update the Access of the Video` request was placed after the publish request, but if it's used, it should come before.